jujugui/static/gui/src/app/app.js

```diff
@@ -693,10 +693,17 @@ YUI.add('juju-gui', function(Y) {
       @param {Boolean} failure Whether the login failed.
     */
     _renderLogin: function(failure) {
+      var msg = 'Find your password with ' +
+                '`juju show-controller --include-passwords`';
```

Comment:
With juju 2.0, the username is now 'admin@local', which is shown by the command.  So maybe say 'Find your username and password'